### PR TITLE
ignore some types when ppretty

### PR DESCRIPTION
**Pull request by menghuu**
ignore some types when ppretty

very useful for showing `pathlib.Path` obj in Config obj

### CODE REVIEW

File: ppretty/ppretty.py [hunk 1]
```diff
@@ -226,3 +233,23 @@ def __init__(self):
     print(ppretty(E(), str_length=19))
 
     print(ppretty({x: x for x in range(10)}))
+
+
+    from pathlib import Path
```
Comment (symonsoft) [inline ppretty/ppretty.py]: pathlib is only in Python 3. I'm still concerned about compatibility with Python 2

File: ppretty/ppretty.py [hunk 2]
```diff
@@ -65,6 +67,12 @@ def inspect_object(current_obj, current_depth, current_width, seq_type_descendan
         if current_obj is None:
             return ['None']
 
+        # Ignored Types
+        if ignore_types is not None and ignore_types is not [None]:
```
Comment (symonsoft) [inline ppretty/ppretty.py]: Why do we need checking for [None]?

File: ppretty/ppretty.py [hunk 3]
```diff
@@ -65,6 +67,12 @@ def inspect_object(current_obj, current_depth, current_width, seq_type_descendan
         if current_obj is None:
             return ['None']
 
+        # Ignored Types
+        if ignore_types is not None and ignore_types is not [None]:
+            for cls in ignore_types:
+                if isinstance(current_obj, cls):
+                    return [str(current_obj)]
```
Comment (symonsoft) [inline ppretty/ppretty.py]: Can be replaced with
```python
if isinstance(current_obj, tuple(ignore_types)):
    return [str(current_obj)]
```

**Review by symonsoft — CHANGES_REQUESTED** [review]

**Comment by menghuu** [comment]
maybe `ignore_subclasses` should be better than `ignore_types` and there are other features about `ignore_xx` and `force_show_xx`. 

maybe keeping this tool simple is a better idea. so I close this, if I think clearfully about those, I will pull request again

feel free to implement those if you think this necessary

thanks for your review

**Pull request closed without merging**